ru/solutions/infrastructure-management/terraform-quickstart.md

```diff
@@ -77,7 +77,7 @@ resource "yandex_compute_instance" "vm-1" {
   }
 
   network_interface {
-    subnet_id = "${yandex_vpc_subnet.subnet-1.id}"
+    subnet_id = "yandex_vpc_subnet.subnet-1.id"
```

Comment:
IIRC, there shouldn't  be any quotation marks, i.e. this line should be `subnet_id = yandex_vpc_subnet.subnet-1.id`

Reply:
True, true... Now I don't think that this PR was a good idea :D 
Fixed